### ISSUE TITLE
DOC: module not found

### ISSUE THREAD

**Issue by arm61** [rhayes777/PyAutoFit#116]
When running the `examples/simple/fit.py` there is a lot 

```
ModuleNotFoundError: No module named 'autofit_workspace'
```

That I fixed manually by removing the `from autofit_workspace.examples.simple ` from the relevant files. I am not sure is this is cause the `PYTHONPATH` environment variable isn't working for me or something? I typically don't touch the `PYTHONPATH` so I am by no means an expert.

**Comment by Jammy2211** [comment]
This looks like a PYTHONPATH issue, in your terminal can you try running:

`export PYTHONPATH=/path/to/autofit_workspace/`

You probably did this when you installed autofit, but your environment resets every time you close the terminal as well as dependent on if you reactivate conda. If that fixes it (as well as the issue https://github.com/rhayes777/PyAutoFit/issues/115) I'll explain how you can make this automatically be set up when your activate the conda environment.

I agree its a bit cumbersome having to mess around with the PYTHONPATH. There are good reason we made this design choice (as it allows a user to update autofit without 'resetting' their workspace), but for those less familiar with editing PYTHONPATH / bash it can trip them up a lot. I think it probably warrants an extra section on the docs.

**Comment by arm61** [comment]
Yeah I already tested exporting the PYTHONPATH immediately before running the `simple/fit.py` with no luck (tried a couple of times now). 

I will investigate how my PYTHONPATH is set up. Cause my brain agrees that this and #115 should work if the PYTHONPATH env var is properly set, I just need to see what is wrong on my end.

**Comment by Jammy2211** [comment]
Are you on MAC, Linux or Windows?

Could you copy and paste what the command line shows if you type $PYTHONPATH, so we can check that its being updated. It may be that instead of export PYTHONPATH we need a set PYTHONPATH or a slightly different syntax depending on the OS..

**Comment by arm61** [comment]
macOS 10.15.5, with a clean conda env of python=3.7. 

echoing the PYTHONPATH returns `/Users/lno76132/Downloads/autofit_workspace` (which is where it should be). and using `set` makes no difference.

**Comment by arm61** [comment]
Gonna try in a virtualenv

**Comment by Jammy2211** [comment]
Try 

`export PYTHONPATH=/Users/lno76132/Downloads/`

Think this is a mistake on our part, albeit I'm sure there was a reason I set things up like this.

**Comment by arm61** [comment]
That's fixed it cheers! Sorry, we probably would have got there faster if I understood PYTHONPATH a bit better. 

I will close #115 but leave this open so the docs can be updated.

**Comment by Jammy2211** [comment]
Pretty sure that is the issue, my apologies! 

I will update the workspace to not include the 'from autofit_workspace' (as opposed to changing the docs regarding the set up of the PYTHONPATH). Do you want me to push you a fresh workspace now or are you ok with your set up? (The truth is it doesn't make much diffeent whichever way its set up, but this way is a bit easier on the eye).

**Comment by arm61** [comment]
Happy with the set up, not to worry!
